sim/druid/moonfire.go

```diff
@@ -68,13 +67,13 @@ func (druid *Druid) registerMoonfireSpell() {
 			SpellSchool: core.SpellSchoolArcane,
 			ProcMask:    core.ProcMaskSpellDamage,
 
-			DamageMultiplierAdditive: 1 +
+			DamageMultiplier: 1 +
```

Comment:
Any particular reason to use the non-additive version here?

Reply:
See above.